### PR TITLE
feat: cart

### PR DESCRIPTION
**Pull request by shashidwivedi**
Cart management page with order items and order pricing

127 0 0 1_5500_pages_shopping-cart html

### CODE REVIEW

**Review by Sameony — APPROVED** [review]
great structure. a minor suggestion, you can give title tag to the cart cards for better explaining the heart and dismiss options

File: styles/main.css [hunk 1]
```diff
@@ -114,3 +114,39 @@ body {
...
+  margin-bottom: var(--space-sm);
+  justify-content: space-between;
+}
+
+.shopping-cart-order .btn-secondary {
+  background-color: var(--color-secondary);
+  width: 100%;
+  height: var(--space-xl);
+  margin-top: var(--space-md);
+}
```
Comment (vaibhav-mahalle) [inline styles/main.css:152]: liked the consistent use of utility classes and variable naming.

File: pages/shopping-cart.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pillow Willow</title>
+    <link
+      rel="stylesheet"
+      href="https://foobar-ui.netlify.app/styles/foobar-ui.min.css"
+    />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="" />
+    <link
```
Comment (shuntzhere) [inline pages/shopping-cart.html:15]: While refactoring, do check if these files are actually required.

**Review by vaibhav-mahalle — APPROVED** [review]
just look into the suggestions by other reviewers. No suggestion from my side.

**Review by coldpigli — APPROVED** [review]
Good use to variables for CSS. Everything is well abstracted. Go ahead and merge

**Comment by shashidwivedi** [comment]
> 

Yes Sameer, as it is using CL. Will update when I will revisit the CL in future

**Review by krishanu2911 — APPROVED** [review]
Well written code no changes required good to be merged👍🏻

File: pages/shopping-cart.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="logo heading-2"><a href="../index.html">Pillow Willow</a></h1>
+
+      <div class="nav flex-row">
```
Comment (shuntzhere) [inline pages/shopping-cart.html:31]: This div can be removed and the classes can be given to the nav tag.

File: pages/shopping-cart.html [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+        </a>
+      </div>
+    </header>
+    <main class="main">
+      <div class="shopping-cart grid-2">
+        <div class="shopping-cart-list grid-col">
+          <div class="shopping-cart-item">
+            <div class="card card_with-shadow card_horizontal">
+              <div class="card-content">
+                <img
```
Comment (shuntzhere) [inline pages/shopping-cart.html:57]: Give an alt title to the image.

**Review by shuntzhere — APPROVED** [review]
Very intuitive class names. Nice use of variables across the file. UI looks very clean too. Suggested a few changes though, do give them a look.

**Comment by shashidwivedi** [comment]
Reopening Pull Request

File: pages/shopping-cart.html (hunk 2 above)
Comment (shashidwivedi) [reply pages/shopping-cart.html:15]: Yes, will check later

File: pages/shopping-cart.html (hunk 4 above)
Comment (shashidwivedi) [reply pages/shopping-cart.html:57]: Yes, will refactor and do

**Pull request merged**